.circleci/config.yml

```diff
@@ -3,15 +3,16 @@ version: 2.1
 orbs:
   snyk: snyk/snyk@1.1.2
 
-jobs:
-  build:
+executors:
+  my-executor:
```

Comment:
This is so we don't have to specify the docker image for **each** job. Just saves us 3 lines.